README.md

```diff
@@ -80,34 +80,37 @@ A number of options can be passed to the wizard as a third argument to customise
 
 `translate` - provide a function for translating validation error codes into usable messages. Previous implementations have used [i18next](https://www.npmjs.com/package/i18next) to do translations.
 `templatePath` - provides the location within `app.get('views')` that templates are stored. Default `pages`.
-`controller` - override the [default controller](./lib/controller.js) for steps without a controller specified.
+* `controller` - The constructor for the controller to be used for request handling. The default is an extension of the [hof-form-controller](https://www.npmjs.com/package/hof-form-controller), which is exported as a `Controller` property of this module. If custom behaviour is required for a particular form step then custom extensions can be defined by passing a list of Behaviours - see [Behaviours](#behaviours)
 `params` - define a suffix for the routes for supporting additional URL parameters.
 
-### Custom Controllers
+### Behaviours
 
-Creating a custom controller:
+Creating a behaviour:
 
-```javascript
-// controller.js
-var util = require('util'),
-    Controller = require('hof-form-wizard').Controller;
-
-function CustomController() {
-  Controller.apply(this, arguments);
-  // extra middleware to log the request
-  this.use(function (req, res, next) {
-    console.log(req.method, req.url);
-    next();
-  });
-}
+```js
+// behaviour-one.js
 
-util.inherits(CustomController)
-
-module.exports = CustomController
+module.exports = SuperClass => class extends SuperClass {
+  constructor() {
+    super();
+    this.use((req, res, next) => {
+      console.log(req.method, req.url);
+      next();
+    });
+  }
```

Comment:
Yeah I agree - I was being lazy and just modified the existing example - I'll change it